<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vertrag</class>
 <widget class="QWidget" name="Vertrag">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neuer Vertrag</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Gesamtpreis:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAbbrechen">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSpeichern">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Speicherrn</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPreis">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ausleihdatum:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kunde:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Fahrzeug:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>RÃ¼ckgabedatum:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxKunde">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFahrzeug">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonKundeDetail">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFahrzeugDetail">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditAusleihe">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditRueckgabe">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dateEditAusleihe</tabstop>
  <tabstop>dateEditRueckgabe</tabstop>
  <tabstop>lineEditPreis</tabstop>
  <tabstop>comboBoxKunde</tabstop>
  <tabstop>buttonKundeDetail</tabstop>
  <tabstop>comboBoxFahrzeug</tabstop>
  <tabstop>buttonFahrzeugDetail</tabstop>
  <tabstop>buttonSpeichern</tabstop>
  <tabstop>buttonAbbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
